nodejs/awsx/apigateway/api.ts

```diff
@@ -606,12 +606,14 @@ function addAPIKeyToSwaggerOperation(swaggerOperation: SwaggerOperation) {
 
 function addAuthorizersToSwagger(
     swagger: SwaggerSpec,
-    authorizers: Authorizer[],
+    authorizers: Authorizer[] | Authorizer,
     apiAuthorizers: Record<string, Authorizer>): Record<string, string[]>[] {
 
     const authRecords: Record<string, string[]>[] = [];
     swagger["securityDefinitions"] = swagger["securityDefinitions"] || {};
 
+    authorizers = authorizers instanceof Array ? authorizers : [authorizers];
```

Comment:
Array.isArray is considered a better/safer check due to the long and sordid history of JS mistakes.